File: app/src/main/java/com/example/toothfairy/util/WearingTimeManager.java
package com.example.toothfairy.util;

import com.example.toothfairy.data.WearingTime;

import org.json.JSONArray;
import org.json.JSONException;
import org.json.JSONObject;

import java.util.ArrayList;

public class WearingTimeManager {

    public static ArrayList<WearingTime> getWearingTimeList(JSONObject jsonObject) throws JSONException {

        JSONArray times = (JSONArray) jsonObject.get("times");

        ArrayList<WearingTime> wearingTimes = new ArrayList<>();

        for (int i = 0; i < times.length(); i++){
            JSONObject time = times.getJSONObject(i);

            Long start  = (Long)time.get("start");
            Long end    = (Long)time.get("end");

            wearingTimes.add(new WearingTime(start, end));
        }

        return wearingTimes;
    }
}

File: app/src/main/java/com/example/toothfairy/util/WearingTimeManager.kt
package com.example.toothfairy.util

import kotlin.Throws
import org.json.JSONException
import com.example.toothfairy.data.WearingTime
import org.json.JSONArray
import org.json.JSONObject
import java.util.ArrayList

object WearingTimeManager {

    @Throws(JSONException::class)
    fun getWearingTimeList(jsonObject: JSONObject): ArrayList<WearingTime> {
        val times = jsonObject["times"] as JSONArray
        val wearingTimes = ArrayList<WearingTime>()

        for (i in 0 until times.length()) {
            val time:JSONObject = times.getJSONObject(i)

            val start   = time["start"] as Long
            val end     = time["end"] as Long

            wearingTimes.add(WearingTime(start, end))
        }

        return wearingTimes
    }
}